test/pathEq.js

```diff
@@ -13,18 +13,23 @@ describe('pathEq', function() {
     };
 
     it('returns true if the path matches the value', function() {
-        assert.ok(R.pathEq('a', 1, obj));
-        assert.ok(R.pathEq('b.ba', '2', obj));
+        assert.strictEqual(R.pathEq(['a'], 1, obj), true);
+        assert.strictEqual(R.pathEq(['b', 'ba'], '2', obj), true);
     });
 
     it('returns false for non matches', function() {
-        assert.ok(!R.pathEq('a', '1', obj));
-        assert.ok(!R.pathEq('b.ba', 2, obj));
+        assert.strictEqual(R.pathEq(['a'], '1', obj), false);
+        assert.strictEqual(R.pathEq(['b', 'ba'], 2, obj), false);
     });
 
     it('returns false for non existing values', function() {
-        assert.ok(!R.pathEq('c', 'foo', obj));
-        assert.ok(!R.pathEq('c.d', 'foo', obj));
+        assert.strictEqual(R.pathEq(['c'], 'foo', obj), false);
+        assert.strictEqual(R.pathEq(['c', 'd'], 'foo', obj), false);
+    });
+
+    it('accepts empty path', function() {
+        assert.strictEqual(R.pathEq([], 42, {a: 1, b: 2}), false);
+        assert.strictEqual(R.pathEq([], undefined, {a: 1, b: 2}), true);
```

Comment:
It depends on whether one thinks the function should *always* return false if the path does not exist.

```javascript
R.pathEq(['x', 'y'], undefined, {x: {y: undefined}});  // => true, indisputably
R.pathEq(['x', 'y'], undefined, {});  // => true? false?
```